docs/appendix/infrastructure-template.md

```diff
@@ -93,18 +81,6 @@ This endpoint can optionally be access controlled with an API key, to enable thi
 export ENABLE_STORAGE_UPLOAD_API_KEY=true
 ```
 
-To deploy this endpoint for Ropsten documents:
-
-```bash
-NETWORK=ropsten npm run deploy:storage -- --stage stg
-```
-
-To deploy this endpoint for Rinkeby documents:
-
-```bash
-NETWORK=rinkeby npm run deploy:storage -- --stage stg
-```
-
```

Comment:
I see that the documentation for testnets has been removed but the mainnet one remained. Is it because this feature still works on mainnet but not on testnets anymore?

Reply:
i see that there is a new URL, will update accordingly

Reply:
Infrastructure template topic is on how user can make use of [oa-functions](https://github.com/Open-Attestation/oa-functions) preconfigured setups to run their own microservices. unless oa-functions (currently broken) is indeed committed by the entire team to maintain it, i suggest we remove this topic entirely and recommend a reference implementation of [tradetrust-functions](https://github.com/TradeTrust/tradetrust-functions) instead.